src/lava/magma/core/process/process.py

```diff
@@ -333,8 +333,10 @@ def run(self,
 
         Parameters
         ----------
-        condition : AbstractRunCondition
-            Specifies for how long to run the Process.
+        condition : Optional[Union[AbstractRunCondition, int]], default=None
```

Comment:
I am a little concerned that it may not be obvious what `condition=10` means when they see this in a tutorial without context. Without knowing what a `RunCondition` is, they may think 10 is some magic number.

This is my main concern with this whole feature. Maybe it can be clarified by renaming the parameter?

Reply:
I would also not prefer to overload. Add another API like run_for_timestep(...) and pipe that through this generic run API. But let's not overload the existing API.